site/_includes/primary-nav-items.html

```diff
@@ -10,6 +10,6 @@
     <li class="training-menu"><a href="/news/">News</a>
     </li>
     <li>
-        <button class="button-default button-green trigger-top-form"> <a href="https://github.com/apache/incubator-metron/releases" target="new">Download </a></button>
+        <button class="button-default button-green trigger-top-form"> <a href="https://dist.apache.org/repos/dist/release/incubator/metron/" target="new">Download </a></button>
```

Comment:
@kylerichardson , I agree, a downloads page is more or less the only way to satisfy the simultaneous requirements of directing the code download through a mirror and the signature download through the dist site.

I would suggest keeping jira METRON-603 open (adding "via Mirror script" to the title) and using it for the correct enhancement later on.  Meanwhile, I have no objection to committing this PR as an improvement to the current situation of having the actual site out of sync with the source code.  We are still sufficiently low volume that we aren't likely to cause problems with Infra.